modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/disruptor/inbound/InboundEventManager.java

```diff
@@ -228,13 +227,13 @@ public void ackReceived(AndesAckData ackData) {
 
             //Tracing message
             if (MessageTracer.isEnabled()) {
-                MessageTracer.trace(ackData.getAcknowledgedMessage().getMessageID(), ackData.getAcknowledgedMessage()
-                        .getDestination(), MessageTracer.ACK_PUBLISHED_TO_DISRUPTOR);
+                MessageTracer.trace(ackData.getIdOfAcknowledgedMessage(), "",
+                        MessageTracer.ACK_PUBLISHED_TO_DISRUPTOR + " Channel = " + ackData.getChannelID());
```

Comment:
There is a format a message is traced. IMO, this is not matching with it. Could we please correct?